chainer/variable.py

```diff
@@ -59,6 +59,10 @@ def __init__(self, data, volatile=False):
            :mod:`cuda` automatically uses this allocator.
 
         """
+        assert(isinstance(data, numpy.ndarray) or
```

Comment:
Add a space between "assert" and the left paren to clarify that assert is not a function.